src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Validator.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Aquifer.Common.Utilities;
+using Aquifer.Data.Enums;
+using FastEndpoints;
+using FluentValidation;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Validator : Validator<Request>
+{
+    public Validator()
+    {
+        RuleFor(x => x.BookCode).NotEmpty();
+        RuleFor(x => x.BookCode)
+            .Must(code => BibleBookCodeUtilities.IdFromCode(code) != BookId.None)
+            .When(x => x.BookCode is not null)
+            .WithMessage(x => "Invalid '{PropertyName}': \"{PropertyValue}\".");
```

Comment:
I don't think you need the `x =>` on any of these. `WithMessage` takes just a string.

Reply:
Thanks. That was an artifact from when I was generating my own property names using `nameof()` and I forgot to remove it after I discovered FluentValidation's templating system.

Fixed [here](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/915feb0d16be4d3215c483cc56b3e3975e5ae498).